tests/foreman/cli/test_computeresource_libvirt.py

```diff
@@ -91,319 +92,332 @@ def valid_update_data():
 
 def invalid_update_data():
     """Random data for invalid update"""
-    return (
+    items = [
         {'new-name': gen_string('utf8', 256)},
         {'new-name': ''},
         {'url': 'invalid url'},
         {'url': ''},
+    ]
+    return {f'{list(item.keys())[0]}=>{list(item.values())[0][0:10]}': item for item in items}
+
+
+@pytest.fixture(scope="module")
+@skip_if_not_set('compute_resources')
+def libvirt_url():
+    return LIBVIRT_RESOURCE_URL % settings.compute_resources.libvirt_hostname
```

Comment:
This is a great candidate for the dynamic formatted dynaconf variables, so that this actually happens on settings file loading and doesn't need to be done in the test. Not a change that's required for this conversion, but worth mentioning as a pattern we can use in the future for this.